examples/java/scenarios/pom.xml

```diff
@@ -35,6 +35,11 @@
       <artifactId>native-assets-client_2.13</artifactId>
       <version>0.3.1</version>
     </dependency>
+    <dependency>
+      <groupId>com.google.protobuf</groupId>
+      <artifactId>protobuf-java-util</artifactId>
+      <version>3.21.12</version>
+    </dependency>
```

Comment:
🤔 is this explicit dependency required?

Reply:
I couldn't find an easy way to change json string to proto Struct without adding a dep